controllers/volumereplicationgroup_controller.go

```diff
@@ -1146,6 +1148,43 @@ func (v *VRGInstance) updateVRGLastGroupSyncTime() {
 	v.instance.Status.LastGroupSyncTime = leastLastSyncTime
 }
 
+func (v *VRGInstance) updateVRGLastGroupSyncDuration() {
+	var maxLastSyncDuration *metav1.Duration
+
+	for _, protectedPVC := range v.instance.Status.ProtectedPVCs {
+		if maxLastSyncDuration == nil {
+			maxLastSyncDuration = protectedPVC.LastSyncDuration
+
+			continue
+		}
+
+		if protectedPVC.LastSyncDuration != nil &&
+			protectedPVC.LastSyncDuration.Duration > maxLastSyncDuration.Duration {
+			maxLastSyncDuration = protectedPVC.LastSyncDuration
+		}
+	}
+
+	v.instance.Status.LastGroupSyncDuration = maxLastSyncDuration
```

Comment:
By the time you get to this line, the value of `maxLastSyncDuration` will equal the value of the first `protectedPVC.LastSyncDuration`. Line 1156 is wrong. 
LastSyncDuration is a pointer, which points to a value somewhere in memory.  maxLastSyncDuration points to the same memory location. And finally `v.instance.Status.LastGroupSyncDuration` points also to the same memory location as well. Have you tested this so that it works? It is hard to believe it works.

Reply:
I have tested this PR. I was able to get the values.